tools/pylib/uvmap.py

```diff
@@ -57,6 +57,31 @@ def Ball(x, y):
         pass
 
 
+def should_be_cut(x, y):
+    a = 0.1
```

Comment:
Can you describe what these parameters do?

Reply:
They were chosen experimentally, but now I've removed 2 unused ones.